### PR TITLE
Fix `nerdctl kill` container with restartPolicy behaviour is inconsistent with docker

### PR DESCRIPTION
**Pull request by yankay**
The PR is to fix the https://github.com/containerd/nerdctl/issues/2315 .

It follows the same way with `nerctl stop`: https://github.com/containerd/nerdctl/blob/main/pkg/containerutil/containerutil.go#L325

### CODE REVIEW

File: pkg/cmd/container/kill.go [hunk 1]
```diff
@@ -65,6 +66,9 @@ func Kill(ctx context.Context, client *containerd.Client, reqs []string, options
 }
 
 func killContainer(ctx context.Context, container containerd.Container, signal syscall.Signal) error {
+	if err := containerutil.UpdateExplicitlyStoppedLabel(ctx, container, true); err != nil {
+		return err
+	}
 	task, err := container.Task(ctx, cio.Load)
```
Comment (fahedouch) [inline pkg/cmd/container/kill.go:72]: is it possible to set also `nerdctl/error` label as we did for `nerdctl stop`  [see](https://github.com/containerd/nerdctl/blob/main/pkg/containerutil/containerutil.go#L320-L324)
Thanks

Comment (yankay) [reply pkg/cmd/container/kill.go:77]: Thanks @fahedouch for the PR review.
It has been changed :-)

**Review by austinvazquez — APPROVED** [review]
Looks good!

**Review by fahedouch — APPROVED** [review]
Thanks

**Comment by yankay** [comment]
Thanks @fahedouch , @austinvazquez 
for the PR review

**Pull request merged**